From 18bb0071413d1b51f5dc5697ab13030a87d4edfb Mon Sep 17 00:00:00 2001
From: mliang2x <mingyuex.liang@intel.com>
Date: Thu, 27 Aug 2020 16:34:39 +0800
Subject: [PATCH] Test case2 for Change build option under driver module

Description: Change build option under BuildOptions.common.EDKII.DXE_SMM_DRIVER, BuildOptions.common.EDKII.SMM_CORE (Add "MSFT:RELEASE_*_*_CC_FLAGS            = /D MDEPKG_NDEBUG") in TestPkg/buildoptions.dsc.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/buildoptions.dsc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/TestPkg/buildoptions.dsc b/TestPkg/buildoptions.dsc
index 9ca9e101ff..bfad814363 100644
--- a/TestPkg/buildoptions.dsc
+++ b/TestPkg/buildoptions.dsc
@@ -28,6 +28,7 @@
 # protection of DXE_SMM_DRIVER/SMM_CORE modules
 [BuildOptions.common.EDKII.DXE_SMM_DRIVER, BuildOptions.common.EDKII.SMM_CORE]
   GCC:*_*_*_DLINK_FLAGS = -z common-page-size=0x1000
+  MSFT:RELEASE_*_*_CC_FLAGS            = /D MDEPKG_NDEBUG
   XCODE:*_*_*_DLINK_FLAGS = -seg1addr 0x1000 -segalign 0x1000
   XCODE:*_*_*_MTOC_FLAGS = -align 0x1000
   CLANGPDB:*_*_*_DLINK_FLAGS = /ALIGN:4096
-- 
2.25.1.windows.1

